Classes/Controller/BackendController.php

```diff
@@ -67,6 +67,19 @@ protected function getPids(array $typeLocations)
         return array_combine($pids, $pids);
     }
 
+    protected function getPageTitles(array $pids): array
+    {
+        foreach ($pids as $pageId) {
+            $row = BackendUtility::getRecord('pages', $pageId, 'title');
```

Comment:
Hey @sypets Thanks for the improvement. Do you check this function? https://api.typo3.org/master/class_t_y_p_o3_1_1_c_m_s_1_1_backend_1_1_utility_1_1_backend_utility.html#ad3d315e94a4d347feb6e88cb4d111f37

This will respect installation specific configs...

Reply:
Thanks. I will check it out and push another change.